sparkle/src/components/Tree.tsx

```diff
@@ -202,11 +202,18 @@ Tree.Item = function ({
 
 interface TreeEmptyProps {
   label: string;
+  onItemClick?: () => void;
 }
 
-Tree.Empty = function ({ label }: TreeEmptyProps) {
+Tree.Empty = function ({ label, onItemClick }: TreeEmptyProps) {
   return (
-    <div className="s-font-regular s-py-1.5 s-pl-6 s-text-sm s-text-muted-foreground">
+    <div
+      className={cn(
+        "s-font-regular s-py-1.5 s-pl-6 s-text-sm s-text-muted-foreground",
+        onItemClick ? "s-cursor-pointer" : ""
+      )}
+      onClick={onItemClick ? onItemClick : undefined}
```

Comment:
`nit`: I guess you can do `onClick={onItemClick}` here ?

Reply:
Done here and in the other existing occurence in the same file! 👍🏻